src/base/static/components/molecules/map-controls.tsx

```diff
@@ -116,19 +116,35 @@ const MapControls: React.FunctionComponent<MapControlProps> = props => {
       <GeolocateControl onViewportChange={props.onViewportChange} />
       <CustomControl
         icon={props.leftSidebarConfig.icon}
-        setLeftSidebarExpanded={props.setLeftSidebarExpanded}
+        onClick={() => props.setLeftSidebarExpanded(true)}
       />
+      {props.isMeasurementToolEnabled && (
+        <CustomControl
+          icon={"fas fa-ruler-combined"}
+          onClick={() =>
+            props.toggleMeasurementToolVisibility(
+              !props.isMeasurementToolVisible,
+            )
+          }
+        />
+      )}
     </div>
   );
 };
 
-const mapDispatchToProps = {
-  setLeftSidebarExpanded,
-};
+const mapDispatchToProps = dispatch => ({
+  setLeftSidebarExpanded: isExpanded =>
+    dispatch(setLeftSidebarExpanded(isExpanded)),
+  toggleMeasurementToolVisibility: isVisible =>
+    dispatch(updateMeasurementToolVisibility(isVisible)),
+});
```

Comment:
nit: This can be written in the object syntax form:

```js
const mapDispatchToProps = {
  setLeftSidebarExpanded,
  updateMeasurementToolVisibility,
}
```